source/src/lccd.cc

```diff
@@ -49,6 +49,8 @@ namespace lccd{
 
 
   LCCDTimeStamp fromSimpleTime( const SimpleTime& st ) {
+    // silence possible warning about unused parameter
+    (void)st;
```

Comment:
```suggestion
  LCCDTimeStamp fromSimpleTime( [[maybe_unused]] const SimpleTime& st ) {
```
Since we have the standardized attribute.

Reply:
This forces C++17 for LCCD and I don't know if this is the case for all the builds, workflows, etc.

Reply:
C++17 has been a requirement for this for quite some time now, so there should be no problem there.